diff -ruN ./sphinx/domains/cpp.py ../Sphinx-1.1.3.p3k/sphinx/domains/cpp.py
--- ./sphinx/domains/cpp.py	2012-01-29 17:49:09.000000000 +0900
+++ ../Sphinx-1.1.3.p3k/sphinx/domains/cpp.py	2012-05-24 00:41:06.524526456 +0900
@@ -162,7 +162,7 @@
         raise NotImplementedError()
 
     def __str__(self):
-        return unicode(self).encode('utf-8')
+        return self.__unicode__()
 
     def __unicode__(self):
         raise NotImplementedError()
